### PR TITLE
Rename function SignIntegrityBlock to be SignWithIntegrityBlock for clarity

### PR DESCRIPTION
**Pull request by sonkkeli**

### CODE REVIEW

File: go/bundle/cmd/sign-bundle/integrityblock.go [hunk 1]
```diff
@@ -24,7 +24,7 @@ func writeOutput(bundleFile io.ReadSeeker, integrityBlockBytes []byte, originalI
 	return nil
 }
 
-func SignIntegrityBlock(privKey crypto.PrivateKey) error {
+func SignWithIntegrityBlock(privKey crypto.PrivateKey) error {
```
Comment (irori) [inline go/bundle/cmd/sign-bundle/integrityblock.go:27]: Maybe it's good idea to add a function comment, since this is a public (upper-cased) function.

Comment (sonkkeli) [reply go/bundle/cmd/sign-bundle/integrityblock.go:30]: Done!

**Comment by sonkkeli** [comment]
PTAL :)

**Review by irori — APPROVED** [review]
lgtm

**Pull request merged**